mmv1/products/compute/VpnGateway.yaml

```diff
@@ -12,21 +12,17 @@
 # limitations under the License.
```

Comment:
See previous comments, this is really a renaming of files between `HaVpnGateway` and `VpnGateway`